en/config/actuators.md

```diff
@@ -0,0 +1,118 @@
+# Actuator Configuration and Testing
+
+After the airframe is setup you should configure the geometry, assign actuators to outputs, and test the actuator response.
+This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+
+:::note
+This view is only enabled if dynamic control allocation is enabled via [SYS_CTRL_ALLOC](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC), which will become the default in future.
+If you enable it, also make sure to select the airframe with [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+It replaces the existing mixer files and allows configuration via parameters instead.
+:::
+
+This is a screenshot for a multicotor:
+
+![Actuators MC (QGC)](../../assets/config/qgc_actuators_mc_aux.png)
+
+
+:::note
+If you find a setting missing (e.g. reversable motors), make sure to enable the **advanced** checkbox in the top right corner.
+:::
+
+## Geometry
+The airframe pre-selects the geometry type, which can provide additional configuration options, such as the motors positions on a multicotor, or the control surface types on a plane.
+The UI reflects that and displays a customized view for each type.
+
+
+### Conventions
+The following sections contain the conventions and explanations for configuring the geometry.
+
+#### Coordinate system
+The coordinate system is NED (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
+Positions are relative to the center of gravity (in meters).
+
+#### Control Surfaces and Servo Direction
+Control surfaces use the following deflection direction convention:
+- horizontal (e.g. Aileron): up = positive deflection
+- vertical (e.g. Rudder): right = positive deflection
+- mixed (e.g. V-Tail): up = positive deflection
+
+![Plane Deflections](../../assets/config/plane_servo_convention.png)
+
+:::note
+If a servo does not move into the expected direction, reverse it via checkbox on the Actuator Output.
+:::
+
+
+### Tilt Servos
```

Comment:
Should we name them "Motor Tilt Servos"? Or just mention somewhere in the beginning that it's about tilting motors, and not control surfaces or anything else.

Reply:
Right, we can do that